Source/WebCore/css/StyleRule.h

```diff
@@ -52,7 +52,7 @@ class StyleSheetContents;
 using CascadeLayerName = Vector<AtomString>;
     
 DECLARE_ALLOCATOR_WITH_HEAP_IDENTIFIER(StyleRuleBase);
-class StyleRuleBase : public RefCounted<StyleRuleBase> {
+class SUPPRESS_REFCOUNTED_WITHOUT_VIRTUAL_DESTRUCTOR StyleRuleBase : public RefCounted<StyleRuleBase> {
```

Comment:
Maybe we wanna add a comment saying that we're doing this as a perf optimization?

Reply:
Okay! I was going to add a comment, then I talked myself out of it.